### DISCUSSION TITLE
.bgeo output files not working with houdini or vray

### DISCUSSION THREAD

**Discussion by Perpetual3134** [InteractiveComputerGraphics/SPlisHSPlasH#280] [Q&A]
The .bgeo output Files are not readable for Houdini or Vray.

Houdini error message: 

Expected array object (near byte offset 32, line 1, column 33)
Backend IO 

Vray error message: 

ply2vrmesh (Converter for .obj or .bgeo to .vrmesh) 

error: Error reading file: [VUtils::BGeoInterface::readFile] 2: Error reading .bgeo file: Magic number not found
[0h 0m 0.0s] [5 MB] error: [VUtils::BGeoInterface::readFile] 2: Error reading .bgeo file: Magic number not found

**Comment by janbender** [comment]
This is true since SPlisHSPlasH is using an older bgeo format than Houdini. The reason for this is that we internally use an older free surface reconstruction software which requires this format. We are already working on an own surface reconstruction solution. However, as long as this is not finished, we still require the old format. 

To solve this, you can try to exchange the partio library with a newer version. This should help.

**Comment by Perpetual3134** [comment]
At the moment my Workaround is to use Splashsurf to generate .obj's and then convert them to .vrmesh

I tried already to change the library, but the error is still the samen. But maybe i did it not in right way.

Is there an internal surface reconstruction in Splishsplash?

**Comment by janbender** [comment]
> Is there an internal surface reconstruction in Splishsplash?

No. splashsurf is the surface reconstruction which we just develop.
